secret-engine/src/main/java/com/sequenceiq/cloudbreak/service/secret/service/SecretService.java

```diff
@@ -30,6 +31,8 @@
 @ConditionalOnBean({VaultKvV2Engine.class, VaultKvV1Engine.class, VaultConfig.class})
 public class SecretService {
 
+    public static final int MAX_VAUL_PATH_LENGTH = 256;
```

Comment:
Nit: MAX_VAULT_PATH_LENGTH (missing T)

Reply:
I won't fix this just to let the checks pass and let unblock users on mowdev. we can fix it later